docker-compose.yml

```diff
@@ -4,14 +4,20 @@ networks:
   todoapp-net:
 services:
   pythonapp:
-    image: todoapp:2.0.0
+    image: todoapp:2.1.0
```

Comment:
The `image` field specifies `todoapp:2.1.0`. Ensure that this image is available in your Docker registry or build context. If it's not, you might need to build it locally or pull it from a registry.